src/admin/index.js

```diff
@@ -620,6 +619,73 @@ module.exports = ({
     return { groups }
   }
 
+  /**
+   * List groups in a broker
+   *
+   * @param {string[]} [groupIds]
+   * @return {Promise<DeleteGroups>}
+   *
+   * @typedef {Object} DeleteGroups
+   * @property {Array<DeleteGroup>} results
+   *
+   * @typedef {Object} DeleteGroup
+   * @property {string} groupId
+   * @property {number} errorCode
```

Comment:
I don't think this is correct anymore. Looking at the code, it looks like it's returning an object with a `results` property containing an array of broker responses. Do you think it would make more sense to return just an array of `DeleteGroup`? Meaning `[{ groupId: 'a', errorCode: 0 }]`.

Reply:
Solved it like that

Reply:
Implemented